.mocharc.yml

```diff
@@ -2,3 +2,4 @@ throw-deprecation: true
 check-leaks: true
 require:
   - '@babel/register'
```

Comment:
If you converted all the tests do we still need Babel here?

Reply:
I'll check. thanks

Reply:
We still need, because the TS files are loading JS source code, and it's Flow so it needs some basic transpilation by Babel.

Reply:
Forgot about that 👍